tests/integration/components/denali-button-test.js

```diff
@@ -186,5 +186,8 @@ module('Integration | Component | denali-button', function (hooks) {
     assert.dom('.button').hasNoText('DenaliButton does not render inner text when `@isLoading` arg is true');
     assert.dom('.button .d-icon').doesNotExist('DenaliButton does not render an icon when `@isLoading` arg is true');
     assert.dom('.button div.loader').exists('DenaliButton renders a loader when `@isLoading` arg is true');
+    assert
+      .dom('.button div')
```

Comment:
Could copy class from above assert
```suggestion
      .dom('.button div.loader')
```

Reply:
Fixed, Please verify

Reply:
I just meant to keep the same selector `'.button div.loader'` so it was asserting that the same element had the class `loader--button`

so

```js
assert
  .dom('.button div.loader')
  .hasClass('loader--button', 'DenaliButton loader element has `loader--button` class when `@isLoading` arg is true');
```